features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,3 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
```

Comment:
Can you add the save_and_open_screenshot command here as well? That is really helpful in debugging.